### PR TITLE
Added Travis ci Integration

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Travis CI Integration
## Description

Added support to run Travis ci build. 
Build will install project dependencies
Run Build Scripts
Run Test Scripts

## Motivation and Context

To Enable CI for any changes which are added to the starter project.

## How Has This Been Tested?

Travis ci setup in forked repo.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "8"
```
Comment (moritzraho) [inline .travis.yml]: Let's use LTS version: 10

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+install:
+  - npm version
+  - npm install
+script:
+  - npm run build
+  - npm test
```
Comment (moritzraho) [inline .travis.yml:13]: Can't we add `npm run e2e` ?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+install:
+  - npm version
+  - npm install
+script:
+  - npm run build
```
Comment (moritzraho) [inline .travis.yml:12]: This is to test building process right?

**Review by moritzraho — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (sandeep-paliwal) [reply .travis.yml]: yes will update this.

File: .travis.yml (hunk 2 above)
Comment (sandeep-paliwal) [reply .travis.yml:13]: e2e has chrome dependency. Will try to get that working in travis ci.

File: .travis.yml (hunk 3 above)
Comment (sandeep-paliwal) [reply .travis.yml:12]: yes. This is to build before we run tests

**Review by moritzraho — APPROVED** [review]

**Comment by moritzraho** [comment]
Awesome, LGTM!

**Comment by moritzraho** [comment]
Awesome, LGTM!

**Pull request merged**